libmscore/stem.cpp

```diff
@@ -105,18 +104,66 @@ void Stem::layout()
                   }
             else {                              // non-TAB
                   // move stem start to note attach point
-                  Note* n  = up() ? chord()->downNote() : chord()->upNote();
-                  if ((up() && !n->mirror()) || (!up() && n->mirror()))
-                        y1 += n->stemUpSE().y();
-                  else
-                        y1 += n->stemDownNW().y();
+                  bool u = up();
+                  Note* n  = u ? chord()->downNote() : chord()->upNote();
+                  bool m = n->mirror();
+                  qreal lw = lineWidthMag();
+                  SymId symId = n->noteHead();
+                  if ((u && !m) || (!u && m)) {
+                        // default offsets for all fonts, fits for most
+                        x  = n->stemUpSE().x() - 0.5 * lw;
+                        y1 = n->stemUpSE().y();
+                        if (symId == SymId::noteheadDoubleWhole) {
+                              x  += n->headWidth();
+                              y1 -= 0.5 * spatium();
+                              }
+                        // some fonts' qlyphs need special treatment, suitable offsets found after lots of trial and error
+                        QString muscicalFont = score()->styleSt(Sid::musicalSymbolFont);
```

Comment:
hate to be "that guy",  but for posterity "musicalFont" will go down in history a little more smoothly ;)

Reply:
_muscical_ would appear to be a misspelling of the word _musical_

Reply:
Outsh!
Thanks, will fix